docs/articles/documentation/test-api/assertions/assertion-api.md

```diff
@@ -208,7 +282,17 @@ Parameter              | Type                                              | Des
 await t.expect('bar').notTypeOf('number', 'string is not a number');
 ```
 
-## Great than
+```js
+import { Selector } from 'testcafe';
+
+fixture `My fixture`;
+
+test('My test', async t => {
+    await t.expect(Selector('#element').getAttribute('attr')).notTypeOf('null');
```

Comment:
there is no such type as `null` in js. `typeof null` returns `object`. You can use `notTypeOf('undefined')` here

Reply:
Is there really no such type?
I've seen it in Helen's examples
https://github.com/DevExpress/testcafe/pull/1783/files#diff-70e36160da834fc09f9ab211e6fd2d9dR251

Reply:
I see now. It's ok. http://chaijs.com/api/assert/#method_typeof